pages/ReviewPage.js

```diff
@@ -0,0 +1,39 @@
+import Header from "@/components/mainHeader/header";
+import BackButton from "@/components/backButton/backButton";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import useLocalStorageState from "use-local-storage-state";
+
+export default function Reviews() {
+  const router = useRouter();
+  const [reviews = []] = useLocalStorageState("review");
```

Comment:
This looks rather complicated. Is this what you want, maybe?
```suggestion
  const [reviews] = useLocalStorageState("review", []);
```